### DISCUSSION TITLE
Resolving SMTP/SMTPS Connectivity Issues in Azure Kubernetes Cluster During Pocketbase Deployment

### DISCUSSION THREAD

**Discussion by skowrons** [pocketbase/pocketbase#3894] [Q&A]
Hello everyone,

We are trying to deploy Pocketbase in Azure within a Kubernetes cluster. However, we are encountering issues connecting to our Mailcow instance and keep receiving a Certificate Error. The same problem occurs with Mailgun and others. We don't have such issues when developing locally, where the connection works smoothly.

According to Microsoft, ports 25/587 are not blocked, and our own services can connect via SMTPs. Has anyone here had similar experiences or problems?

**Comment by ganigeorgiev** [comment]
Could you paste the exact error that is being returned?

If it is something like "signed by unknown authority" then most likely your PocketBase container doesn't have the root certificates installed. If that's the case make sure that the `ca-certificates` package is installed (_the name could differ depending on the used distro_).
